picard/util/__init__.py

```diff
@@ -383,21 +383,19 @@ def album_artist_from_path(filename, album, artist):
     return album, artist
 
 
-def build_qurl(host, port=80, path=None, mblogin=False, queryargs=None):
+def build_qurl(host, port=80, path=None, queryargs=None):
     """
     Builds and returns a QUrl object from `host`, `port` and `path` and
     automatically enables HTTPS if necessary.
 
-    Setting `mblogin` to True forces HTTPS on MusicBrainz' servers.
-
     Encoded query arguments can be provided in `queryargs`, a
     dictionary mapping field names to values.
     """
     url = QtCore.QUrl()
     url.setHost(host)
     url.setPort(port)
-    if (# Login is required and we're contacting an MB server
-        (mblogin and host in MUSICBRAINZ_SERVERS and port == 80) or
+    if (# We're contacting a MusicBrainz server
+        (host in MUSICBRAINZ_SERVERS and port == 80) or
```

Comment:
I removed the `port` check with https://github.com/metabrainz/picard/pull/600/commits/e85febfd2357b558b3f83d41e2459b062ff29d46.

The second part of your comment feels like something beyond this PR. Note that `build_qurl()` is used for all requests, not just for {beta.,musicbrainz.org} ones.